packages/api/src/routes/webhook/apple.ts

```diff
@@ -38,7 +38,11 @@ routes.post(
 
     if (payload.error) {
       capture.error(new Error(payload.error), {
-        extra: { metriportUserId, context: `webhook.appleError` },
+        extra: {
+          metriportUserId,
+          context: `webhook.appleError`,
+          ...(payload.extra ? JSON.parse(payload.extra) : {}),
```

Comment:
Wouldn't extra already be parsed? Or are we sending extra "stringified" from Apple?

Reply:
Yea its extra stringified